scripts/monitoring/cron-send-os-master-metrics.py

```diff
@@ -372,6 +373,34 @@ def nodes_not_ready(self):
         self.zagg_sender.add_zabbix_keys(
             {'openshift.master.nodesnotschedulable.count' : len(nodes_not_schedulable)})
 
+
+    def nodes_not_labeled(self):
+        """ check the nodes in the cluster that are not labeled
+            Note: This check only searches for nodes with no label keys set"""
+
+        print "\nPerforming nodes not labeled check..."
+
+        response = self.ora.get('/api/v1/nodes')
+
+        nodes_not_labeled = []
+        nodes_labeled = []
+        error_found = 1
+
+        for n in response['items']:
```

Comment:
@joelddiaz yeah I was looking to do something like that but was having issues, I don't think I was looking for 'labels' in the right place.

Tested your code and it is spot on!